draft-ietf-wish-whep.md

```diff
@@ -633,6 +633,25 @@ data: /DA8AAAAAAAAAP///wb+06ACpQAmAiRDVUVJAACcHX//AACky4AMEERJU0NZTVdGMDQ1MjAwME
 ~~~~~
 {: title="scte35 example event"}
 
+### Events over WebRTC datachannel
+
+In some architectures sending events over a long polled HTTP request from a server that isn't the media server may be costly, such servers may support events over the webRTC datachannel.
```

Comment:
the http requests are served by the WHEP resource (which may not be collocated with the media server)

Reply:
Yes, that's the point I'm making (badly) the event originates from the media server but has to go via the WHEP resource server (HTTP) to get to the client when there is a more direct path to hand!